src/runtime-rs/crates/runtimes/common/src/sandbox.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use anyhow::Result;
+use async_trait::async_trait;
+
+#[async_trait]
+pub trait Sandbox: Send + Sync {
```

Comment:
Suggest renaming `SandboxManager` to be consistent with `ContainerManager` (and facilitate IDE navigation when you have both the agent and the runtime)

Reply:
Thanks for your review, I think there is a little difference, the container needs to manage multiple so use `ContainerManager` but there is only one sandbox, so I think it's better to use `Sandbox`.